interceptors/logging/interceptors.go

```diff
@@ -141,8 +141,15 @@ func reportable(logger Logger, opts *options) interceptors.CommonReportableFunc
 			kind = KindClientFieldValue
 		}
 
-		// Field dups from context override the common fields.
-		fields := newCommonFields(kind, c).WithUnique(ExtractFields(ctx))
+		fields := Fields{}
+		if opts.disableGrpcLogFields != nil {
+			fields = disableCommonLoggingFields(kind, c, opts.disableGrpcLogFields)
+		} else {
+			// Field dups from context override the common fields.
+			fields = newCommonFields(kind, c)
+		}
```

Comment:
You can rewrite this to avoid the else
```suggestion
		// Field duplicates from the context may override the common fields.
		fields := newCommonFields(kind, c)
		if opts.disableGrpcLogFields != nil {
			fields = disableCommonLoggingFields(kind, c, opts.disableGrpcLogFields)
		}
```